warehouse/accounts/services.py

```diff
@@ -211,6 +227,14 @@ def update_user(self, user_id, **changes):
 
         return user
 
+    def update_two_factor(self, user_id, **changes):
+        two_factor = self.get_two_factor(user_id)
```

Comment:
This can return ``None``, which is going to generate an error like ``AttributeError: "NoneType" object has no attribute "whatever"``. We should probably just make that an explicit error with a meaningful message instead.

Reply:
Sounds good to me! Is this just a matter of throwing a better exception, or is there a different way to propagate an error from here? Many of the other service calls (e.g. `add_email`, `update_user`) appear to ignore the case where `get_user` returns `None`, but maybe I'm missing some implicit checking?